### PR TITLE
replaced service chain readme

### PR DESCRIPTION
**Pull request by terrikwak**
replaced service chain readme

### CODE REVIEW

**Comment by nohkwak** [comment]
Terri
When using the name "Service Chain" or "ServiceChain", our cell members recommended using it continuously without a space in the middle, such as "ServiceChain".
It may be inconvenient to read as “ServiceChain”, but since there is a term called SideChain in Ethereum, it seems to match this term, and it seems to be conceptually easy to understand, and it will be emphasized that it is klaytn's unique solution.
Thank you.

File: docs/node/service-chain/references/README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# Reference Manuals <a id="reference-manuals"></a>
+# Service Chain <a id="service-chain"></a>
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:1]:
```suggestion
# ServiceChain <a id="service-chain"></a>
```

File: docs/node/service-chain/references/README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+## Service Chain Overview <a id="service-chain-overview"></a>
+
+Klaytn Service Chain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn Service Chain provides the following features:
+
+- Immediate finality
+- Cross-chain token transfer
+- Data anchoring to the main chain for data integrity
+- Multi-sig bridge contract to meet enterprise-level security requirements
+
+![](../../klaytn/images/sc_serviceChains.png)
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:18]:
```suggestion
![](./images/sc_serviceChains.png)
```
I request changing the image path.

File: docs/node/service-chain/references/README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
-# Reference Manuals <a id="reference-manuals"></a>
+# Service Chain <a id="service-chain"></a>
 
+## Intended Audience <a id="intended-audience"></a>
+
+- Companies that want to build blockchains for Metaverse, GameFi, and NFTs
+- dApp developers who need high TPS, minimal transaction fees, and data privacy.  
+- Anyone who wants to build a local private network or a ledger database for testing.
+
+## Service Chain Overview <a id="service-chain-overview"></a>
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:9]:
```suggestion
## ServiceChain Overview <a id="service-chain-overview"></a>
```

File: docs/node/service-chain/references/README.md [hunk 4]
```diff
@@ -1,2 +1,21 @@
...
+
+Klaytn Service Chain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn Service Chain provides the following features:
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:11]:
```suggestion
Klaytn ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:
```

File: docs/node/service-chain/references/README.md [hunk 5]
```diff
@@ -1,2 +1,21 @@
...
+
+- Immediate finality
+- Cross-chain token transfer
+- Data anchoring to the main chain for data integrity
+- Multi-sig bridge contract to meet enterprise-level security requirements
+
+![](../../klaytn/images/sc_serviceChains.png)
+
+
+Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the Service Chain.
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:21]:
```suggestion
Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the ServiceChain.
```

File: docs/node/service-chain/references/README.md [hunk 6]
```diff
@@ -1,2 +1,21 @@
...
+
+- Immediate finality
+- Cross-chain token transfer
+- Data anchoring to the main chain for data integrity
+- Multi-sig bridge contract to meet enterprise-level security requirements
+
+![](./images/sc-overview.png)
+
+
+Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the ServiceChain.
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:21]:
```suggestion
Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the ServiceChain. And the following videos will help you understand Klaytn ServiceChain.

[Horizontal Scaling through Service Chain in Klaytn | TXGX 2019](https://www.youtube.com/watch?v=8yQc5FQysJc)
[High Availability Architecture of Klaytn Service Chain | TXGX 2019](https://www.youtube.com/watch?v=HcdhWtXPuR0)
```

**Comment by nohkwak** [comment]
sc-overview

There is a feedback that the file size is large, so please replace the file.
Thank you always.

**Comment by terrikwak** [comment]
@nohkwak  I changed the image.. can I merge this?

**Comment by nohkwak** [comment]
Another change request was made because the image resolution was low. Please change it to the file I am attaching now, and please merge it.
Sorry for the inconvenience.
Thank you so much. 

sc-overview
.

**Comment by terrikwak** [comment]
@nohkwak  no problem :) could you approve? I changed the image

**Review by nohkwak — APPROVED** [review]

File: docs/node/service-chain/references/README.md [hunk 7]
```diff
@@ -1,2 +1,24 @@
...
+- Companies that want to build blockchains for Metaverse, GameFi, and NFTs
+- dApp developers who need high TPS, minimal transaction fees, and data privacy.  
+- Anyone who wants to build a local private network or a ledger database for testing.
+
+## ServiceChain Overview <a id="service-chain-overview"></a>
+
+Klaytn ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:
+
+- Immediate finality
+- Cross-chain token transfer
```
Comment (kjhman21) [inline docs/node/service-chain/references/README.md:14]: Hmm can it be misunderstood to readers that ServiceChain feature can transfer tokens between any other blockchains?

Token transfer between ServiceChains?
Token transfer between Klaytn chains?

File: docs/node/service-chain/references/README.md [hunk 8]
```diff
@@ -1,2 +1,24 @@
...
 
+## Intended Audience <a id="intended-audience"></a>
+
+- Companies that want to build blockchains for Metaverse, GameFi, and NFTs
+- dApp developers who need high TPS, minimal transaction fees, and data privacy.  
+- Anyone who wants to build a local private network or a ledger database for testing.
+
+## ServiceChain Overview <a id="service-chain-overview"></a>
+
+Klaytn ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:
```
Comment (nohkwak) [inline docs/node/service-chain/references/README.md:11]:
```suggestion
ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:
```

File: docs/node/service-chain/references/README.md (hunk 7 above)
Comment (nohkwak) [reply docs/node/service-chain/references/README.md:14]: I suggest the revised phrase in the review below.

Comment (terrikwak) [inline docs/node/service-chain/references/README.md:14]:
```suggestion
- Token transfer between Klaytn chains
```

**Comment by terrikwak** [comment]
@kjhman21 @nohkwak can you have a final look? i applied your suggestion and i also changed the image

**Review by kjhman21 — APPROVED** [review]
Good to go! @terrikwak Thanks for the image!

**Comment by terrikwak** [comment]
@neoofklaytn @dcground  can you approve this? It looks like you need 4 approvals to merge this PR.. 😰

File: docs/node/service-chain/references/README.md (hunk 7 above)
Comment (nohkwak) [reply docs/node/service-chain/references/README.md]: @kjhman21 
I am sorry 
I understood it as a way to collaborate like document editing in Google Docs,
In the future, we will leave it without resolving.

**Comment by nohkwak** [comment]
@terrikwak 
Thanks a lot. 
LGTM.

**Pull request merged**